src/test/java/uk/gov/justice/hmpps/datacompliance/client/duplicate/detection/AwsAthenaDuplicateDetectionClientTest.java

```diff
@@ -0,0 +1,154 @@
+package uk.gov.justice.hmpps.datacompliance.client.duplicate.detection;
```

Comment:
I'm guessing there is no way to integration test this, I'm guessing there is no localstack config of Athena?

Reply:
Unfortunately not, unless we have access to the pro version of Localstack: https://github.com/localstack/localstack#overview

Reply:
Gosh, it is actually really cheap, £15 per month per CI/Developer
It might be worth actually perusing that, not sure who we ask.